### DISCUSSION TITLE
2.0 Feature Discussion

### DISCUSSION THREAD

**Discussion by jdev082** [CatalystDevOrg/Catalyst#129] [General]
Discuss features for the Scottish Fold Feature Selection.

**Comment by hello-smile6** [comment]
Maybe automatically open disk images in v86 if the user wants?

**Reply by imdevarsh** [reply]
Why?

**Reply by jdev082** [reply]
@hello-smile6 again useless features.

**Reply by imdevarsh** [reply]
@JaydenDev at least get more clarification from @hello-smile6 
we might be understanding it wrong
we can't call everything useless, I don't understand it yet

**Reply by hello-smile6** [reply]
> @hello-smile6 again useless features.

Why? For people like me, it'd be amazing! Could you maybe do plugins?

**Reply by imdevarsh** [reply]
Plugins, as in userscripts? #127 may do it in the future.

**Reply by hello-smile6** [reply]
> Plugins, as in userscripts? #127 may do it in the future.

Not that. I mean, like Adobe Flash. Except without the security vulnerabilities.

**Reply by jdev082** [reply]
@hello-smile6 Adobe flash is dead, and an unneeded features as well!

**Reply by hello-smile6** [reply]
> @hello-smile6 Adobe flash is dead, and an unneeded features as well!

LOL. 

> like

**Reply by imdevarsh** [reply]
do you mean wasm? idk if catalyst has wasm

**Reply by jdev082** [reply]
@webdev03 It doesn't and won't

**Reply by imdevarsh** [reply]
Why not? Quite a few apps use WebAssembly nowadays.

**Reply by jdev082** [reply]
@webdev03 web app support is only partially existant as well. But right now the focus is user scripts/styles and UI.

**Reply by hello-smile6** [reply]
> do you mean wasm? idk if catalyst has wasm

Nope, I mean a plugin. Also, whether the Chromium build you're building Catalyst with has WASM is the real question.


> @webdev03 web app support is only partially existant as well. But right now the focus is user scripts/styles and UI.

That's actually managed for you by Electron.

**Reply by hello-smile6** [reply]
@JaydenDev Did you see the above comment?

**Reply by jdev082** [reply]
@hello-smile6 2.0 was canceled so I don't know what your talking about.
